FEXCore/include/FEXCore/Utils/StringUtils.h

```diff
@@ -28,4 +28,41 @@ namespace FEXCore::StringUtils {
     return RightTrim(LeftTrim(std::move(String), TrimTokens), TrimTokens);
   }
 
+  namespace StringHasher {
```

Comment:
This doesn't seem to be related to the other changes. Did you mean to include any other patches?

If not, reviewing this along with code that actually uses it seems more sensible.

Reply:
This is used in GdbServer.cpp

Reply:
Ah, yes... "Large diffs are not rendered by default" got me once again :(